lib/appsignal.rb

```diff
@@ -301,6 +294,15 @@ def start_stdout_logger
         "appsignal: #{msg}\n"
```

Comment:
This is because Heroku already appends timestamps to sdout logs. Not sure how this works on other platforms.